modules/ROOT/pages/batch-processing-concept.adoc

```diff
@@ -112,16 +112,19 @@ This first phase is implicit. During this phase, the runtime performs all the "b
 Mule exposes the batch job instance ID through the `batchJobInstanceId` variable. This variable is available in every step and the on-complete phase.
 +
 . Mule creates a persistent queue and associates it with the new batch job instance.
++
+Queue names are prefixed with `BSQ`, and you can see them in Anypoint Runtime Manager:
++
+image:mruntime-batch-bsq.png[Batch step queue transaction log]
+
 . For each item generated by the splitter, Mule creates a record and stores it in the queue. This activity is "all or nothing" – Mule either successfully generates and queues a record for every item, or the whole message fails during this phase.
 . Mule presents the batch job instance, with all its queued-up records, to the first batch step for processing.
 
-After this phase completes, the flow will continue.
-
-The next phase, "Process," is asynchronous, meaning that the flow will not wait for the batch job to finish processing all records.
+After this phase completes, the flow continues its execution without waiting for the batch job to finish processing all records. This behavior occurs because the next phase, _Process_, is asynchronous. 
 
 === Process
 
-During the "Process" phase, the runtime begins processing the records in the batch asynchronously. Each record moves through the processors in the first batch step, then is sent back to the original queue while it waits to be processed by the second batch step and so on until every record has passed through every batch step. Only one queue exists, and records are picked out of it for each batch step, processed, and then sent back to it; each record keeps track of what stages it has been processed through while it sits on this queue. Note that a batch job instance does not wait for all its queued records to finish processing in one batch step before pushing any of them to the next batch step. Queues are persistent.
+During the _Process_ phase, the runtime begins processing the records in the batch asynchronously. Each record moves through the processors in the first batch step, then is sent back to the original queue while it waits to be processed by the second batch step and so on until every record has passed through every batch step. Only one queue exists, and records are picked out of it for each batch step, processed, and then sent back to it; each record keeps track of what stages it has been processed through while it sits on this queue. Note that a batch job instance does not wait for all its queued records to finish processing in one batch step before pushing any of them to the next batch step. Queues are persistent.
```

Comment:
@marianogonzalez If a job instance can create additional queues, should we clarify that here were it is mentioned that `Only one queue exists` ?

Reply:
per each job instance, there's only one queue which is used to hold the records and to make them flow across the steps... However, additional queues can be created for other internal purposes.